### DISCUSSION TITLE
How to set up the language (for Date format) in a quarto document?

### DISCUSSION THREAD

**Discussion by LuisLauM** [quarto-dev/quarto-cli#9875] [Q&A]
### Description

When using `Date` objects, there are some options that are influenced by the language set on the PC where the command is executed. For example, if one runs this in-line chunk: `r format(x = Sys.Date(), format = “%B, %Y”)`, the month name will be defined according to the language of the PC where that chunk (the qmd file) is run. 

So, is there any way to define the language (in the yaml header maybe) in which a .qmd file should be compiled so that the result is the same (e.g. month names in English) regardless of the language set on the PC where it is executed?

**Comment by mcanouil** [comment]
What is the link with Quarto? You question seems to only involve the use of R and locales.
As you would do in any R script, you have to configure your environment using options, environment variables, etc.
Depending on the method, you can use parameters but fundamentally, it's independent of the language set in QUarto.

FYI, Quarto does not use R for dates, see <https://quarto.org/docs/reference/dates.html>.

**Comment by cderv** [comment]
The internationalization works in Quarto through the `lang` field. The date parsing will be sensible to it. Though the format syntax is different than R

````markdown
---
title: "dash-test"
format: html
date: today
date-format: "MMM, YYYY"
lang: fr
engine: knitr
---

# Setting the date

Today's date in french is `r format(x = Sys.Date(), format = "%B, %Y")`
````

![image](https://github.com/quarto-dev/quarto-cli/assets/6791940/f21f3038-de6b-4e6a-957c-3cc3fbc27147)

Quarto does not look at your R locale or the computer locale. default `lang` is english.
